crypto/fipsmodule/ec/p224-64.c

```diff
@@ -204,7 +199,7 @@ static void p224_felem_to_bin28(uint8_t out[28], const p224_felem in) {
 }
 
 static void p224_generic_to_felem(p224_felem out, const EC_FELEM *in) {
-  p224_bin28_to_felem(out, in->bytes);
```

Comment:
There were some changes (deletions) before this one in the original commit, why we didn't take them?

Reply:
The deletion in the original commit were removing `p224_bin28_to_felem` and `p224_load_u64`. But, because I didn't take the change to `p224_generic_to_felem`, those functions are still in use.

Reply:
Why we didn't take the change to `p224_generic_to_felem`?

Reply:
Fixed in latest update.